docs/known_issues.md

```diff
@@ -83,7 +83,7 @@ use Vich\UploaderBundle\Mapping\Annotation as Vich;
 class Product
 {
     /**
-     * @Vich\UploadableField(mapping="product_image", fileNameProperty="imageName")
+     * @Vich\UploadableField(mapping="products", fileNameProperty="imageName")
```

Comment:
Here you're using annotation.

If you want to be consistent with the config, you should use attribute everywhere